src/symbol/placement.js

```diff
@@ -763,22 +772,27 @@ export class Placement {
             const hasText = numHorizontalGlyphVertices > 0 || numVerticalGlyphVertices > 0;
             const hasIcon = symbolInstance.numIconVertices > 0;
 
+            const placedOrientation = this.placedOrientations[symbolInstance.crossTileID];
+            const verticalHidden = (placedOrientation === WritingMode.horizontal || placedOrientation === WritingMode.horizontalOnly) ? 1 : 0;
+            const horizontalHidden = placedOrientation === WritingMode.vertical ? 1 : 0;
+
```

Comment:
Do we need an assertion here that both `verticalHidden` and `horizontalHidden` are not `0`.
`horizontallHidden` can also be `!verticalHidden`, right? since we have 3 writing modes

Reply:
Ah ok!, could you leave a comment about this?